backend/api/Controllers/ReturnToHomeController.cs

```diff
@@ -10,21 +11,22 @@ namespace Api.Controllers
     [Route("return-to-home")]
     public class ReturnToHomeController(
         ILogger<RobotController> logger,
-        IReturnToHomeService returnToHomeService,
-        IRobotService robotService
+        IRobotService robotService,
+        IIsarService isarService
     ) : ControllerBase
     {
         /// <summary>
         ///     Sends the robots to their home.
         /// </summary>
         [HttpPost("schedule-return-to-home/{robotId}")]
         [Authorize(Roles = Role.User)]
-        [ProducesResponseType(typeof(MissionRun), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status409Conflict)]
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-        public async Task<ActionResult<IList<MissionRun>>> ScheduleReturnToHomeMission(
+        public async Task<ActionResult<IsarMission>> ScheduleReturnToHomeMission(
```

Comment:
Why do you want to return a IsarMission here? I dont the the users of Flotilla should ever have to concern themselves with IsarMissions

Reply:
Agreed, removed it